pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -250,17 +268,20 @@ func ingest(ctx context.Context, execCtx sql.JobExecContext, ingestionJob *jobs.
 		log.Infof(ctx,
 			"starting to revert to the specified cutover timestamp for stream ingestion job %d",
 			ingestionJob.ID())
+		updateRunningStatus(ctx, ingestionJob, "starting to cut over to the given timestamp")
 		if err = revertToCutoverTimestamp(ctx, execCtx, ingestionJob.ID()); err != nil {
 			return err
 		}
 
 		log.Infof(ctx, "activating destination tenant %d", details.NewTenantID)
+		updateRunningStatus(ctx, ingestionJob, "activating destination tenant")
```

Comment:
I think we can elide this one. I think it is reasonable that "cutover" includes the tenant activation.